ui/src/adapters/env.ts

```diff
@@ -33,6 +35,7 @@ export const envParser = getStrictParser<EnvInput, Env>()(
         VITE_API_URL,
         VITE_API_USERNAME,
         VITE_BLOCK_EXPLORER_URL,
+        VITE_BUILD_TAG,
```

Comment:
It's missing in https://raw.githubusercontent.com/0xPolygonID/issuer-node/PID-690-optional-build-tag/ui/.env.sample

Reply:
Oops! Thanks, fixed.